source/reference/kubectl-minio-plugin/kubectl-minio-tenant-create.rst

```diff
@@ -0,0 +1,199 @@
+
+   Ensure that the specified number of :mc-cmd:`~kubectl minio tenant create --servers` does *not* exceed the number of nodes in the Kubernetes cluster. 
+   MinIO strongly recommends sizing the cluster to have one node per MinIO server.
+
+.. mc-cmd:: --volumes
+   :required:
+   :fullpath:
+
+   Total number of volumes to use in the MinIO tenant.
+   
+   :mc-cmd:`kubectl minio tenant create` generates one :kube-docs:`Persistent Volume Claim (PVC)  <concepts/storage/persistent-volumes/#persistentvolumeclaims>` for each volume. 
+
+   The number of volumes affects both the requested storage of each ``PVC`` *and* the number of ``PVC`` to associate to each MinIO Pod in the cluster:
+
+   - The command :mc:`kubectl minio` divides the :mc-cmd:`~kubectl minio tenant create --capacity` by the number of volumes to determine the amount of ``resources.requests.storage`` to set for each ``PVC``.
+
+   - :mc:`kubectl minio` determines the number of ``PVC`` to associate to each ``minio`` server by dividing :mc-cmd:`~kubectl minio tenant create --volumes` by         :mc-cmd:`~kubectl minio tenant create --servers`.
+
+   The command generates each ``PVC`` with Pod-specific selectors, such that each Pod only uses ``PV`` that are locally-attached to the node running that Pod.
+
+   If the specified number of volumes exceeds the number of unbound ``PV`` available on the cluster, :mc:`kubectl minio tenant create` hangs and waits until the required ``PV`` exist.
+
+.. mc-cmd:: --enable-host-sharing
+   :optional:
+   :fullpath:
+
+   .. important::
+   
+      To be used in testing environments only.
+      This flag is **not** supported in production environments.
+   
+   Disable pod anti-affinity to allow co-location of pods on a single node.
+
+.. mc-cmd:: --image
+   :optional:
+   :fullpath:
+
+   MinIO image to use for the tenant.
+   Defaults to the latest minio release.
+
+.. mc-cmd:: --image-pull-secret
+   :optional:
+
+   The image secret to use for pulling MinIO.
+
+.. mc-cmd:: --kes-config
+   :optional:
+   :fullpath:
+
+   The name of the Kubernetes Secret which contains the MinIO Key Encryption Service (KES) configuration. 
+   Required for enabling Server Side Encryption of objects (SSE-S3).
+
+   For more, see the `Github documentation <https://github.com/minio/operator/blob/master/examples/kes-secret.yaml>`__. 
+
+.. mc-cmd:: --namespace
+   :optional:
+   :fullpath:
+
+   The namespace in which to create the MinIO Tenant and its associated resources. 
+      
+   MinIO supports exactly *one* MinIO Tenant per namespace. 
+   Create a unique namespace for each MinIO Tenant deployed into the cluster.
+
+   Defaults to ``minio``.
+   
+.. mc-cmd:: --output
+   :optional:
+   :fullpath:
+
+   Outputs the generated ``YAML``-formatted specification objects to ``STDOUT`` for further customization. 
+      
+   :mc-cmd:`~kubectl minio tenant create --output` does **not** create the MinIO Tenant. 
```

Comment:
the description could be consistent with the help text:
`dry run this command and generate requisite yaml`

Reply:
Updated, though I dropped `requisite`.